### ISSUE TITLE
feat: File Based Provider Support

### ISSUE THREAD

**Issue by crhuber** [containous/traefik-helm-chart#21]
We currently require routing to services that are outside Kubernetes. Without File based provider we cant route to them, so it would be nice if in the helm chart you can pass args to enable it

**Comment by dduportal** [comment]
Another use case: Specify a TLS wildcard certificate from a file/secret only available in Traefik's namespace (if you have 20 subdomains for 20 applications in their own 20 namespaces, you might not want to duplicate 20 times the wildcard certificate)

**Comment by dduportal** [comment]
The Pull Request #34 will solve half of this issue, by allowing to enable the file provider with

```
helm install --name=traefik --set=additionalArguments="{--providers.file.filename=/dyn.yml}" .
```

for instance

**Comment by mmatur** [comment]
Fixed by #34

**Comment by igorbrigadir** [comment]
If anyone else is looking here - this is a good working example: https://traefik.io/blog/install-and-configure-traefik-with-helm/ the key was `--providers.file.filename=/data/traefik-config.yaml` and mounting a `type: configMap` volume where you can add any file based configs for middlewares, routers etc.
